guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: guide
+title: Getting to know your users
+description: How best to understand and develop knowledge about your users.
+parent: Onboarding
+nav_order: 1
+permalink: /guide/onboarding/getting-to-know-your-users/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/getting-to-know-your-users.svg"
```

Comment:
Really like the 3D-rendered style of the image, but I don't understand how the image content relates to the page content. Can you please share your thinking around this?

Reply:
Not sure if these banner images need to be super closely linked to the content of the page, I take the image is a bit abstract but I don't think it takes away from much IMO. Although maybe finding something with more of a middle ground might work better but probably not a game-changer.

Reply:
OK, so it's basically a decorative image?

Reply:
Yeah, think it's a deal-breaker?